api/android/api/src/main/java/org/nnsuite/nnstreamer/TensorsInfo.java

```diff
@@ -57,7 +64,7 @@ public int getTensorsCount() {
      * @throws IndexOutOfBoundsException when the maximum number of tensors in the list
      * @throws IllegalArgumentException if given param is null or invalid
      */
-    public void addTensorInfo(int type, @NonNull int[] dimension) {
+    public void addTensorInfo(NNStreamer.TensorType type, @NonNull int[] dimension) {
```

Comment:
With the current implementation, `addTensorInfo` can make make a TensorData inconsistent, as the TensorInfo inside the TensorData can be modified without modifying the memory allocated by TensorData. Should that be synced (in case its already synced), or its responsibility of the caller? This is because addTensorData keeps TensorData and TensorInfo synced.
The cases where TensorData and TensorInfo are not synced (TensorData allocaed without TensorInfo) are ignored.

Reply:
I supposed that developer should handle proper data conversion between TensorsInfo and TensorsData class. (no-sync after allocating data from TensorsInfo object)

Reply:
Or, is it necessary to implement cloneable interface?

Reply:
Implemented clone() method in TensorsInfo class to prevent the meta-info change after allocaing the byte-buffer.